libraries/networking/src/NetworkingConstants.h

```diff
@@ -30,6 +30,9 @@ namespace NetworkingConstants {
 
     // Web Engine requests to this parent domain have an account authorization header added
     const QString AUTH_HOSTNAME_BASE = "highfidelity.com";
+    
+    // Use a custom User-Agent to avoid ModSecurity filtering, e.g. by hosting providers.
+    const QByteArray VIRCADIA_USER_AGENT = "Mozilla/5.0 (HighFidelityInterface)";
```

Comment:
Can the value be changed to, say, ""Mozilla/5.0 (VircadiaInterface)"?

There are some other places where "HighFidelityInterface" are used, which may need to be addressed:
![HighFidelityInterface](https://user-images.githubusercontent.com/7455448/86972157-907cba00-c1c6-11ea-9c10-16ae870a3588.png)

Reply:
It could potentially, I was wondering about user agents being used to connect to the metaverse API, I could have sworn there's something you need to pass the user agent to in order to get through... but wasn't sure.

Reply:
Try making the changes and see what breaks (e.g., does logging in to HiFi metaverse still work)?

Reply:
It appears to all work fine. I'll push the update.

Reply:
There are still some other places it's used. Should these also be changed?

Reply:
I changed the most obvious other places... appears to still work. The last obvious places are for things like the Assignment Client, but that one for example uses a DEFINE and I'm not sure why, or if it can change. So I'll leave that alone for now.